internal/uvm/start.go

```diff
@@ -115,6 +118,38 @@ func parseLogrus(vmid string) func(r io.Reader) {
 	}
 }
 
+// When using an external GCS connection it is necessary to send a ModifySettings request
+// for HvSockt so that the GCS can setup some registry keys that are required for running
+// containers inside the UVM. In non external GCS connection scenarios this is done by the
+// HCS immediately after the GCS connection is done. Since, we are using the external GCS
+// connection we should do that setup here after we connect with the GCS.
+// This only applies for WCOW
+func (uvm *UtilityVM) configureHvSocketForGCS(ctx context.Context) (err error) {
+	if uvm.OS() != "windows" {
+		return nil
+	}
+
+	hvsocketAddress := &hcsschema.HvSocketAddress{
+		LocalAddress:  uvm.runtimeID.String(),
+		ParentAddress: gcs.WindowsGcsHvHostID.String(),
+	}
+
+	conSetupReq := &hcsschema.ModifySettingRequest{
+		RequestType: requesttype.Update,
```

Comment:
I think you can leave off this `RequestType` line here. The modify operation will ignore the top level request if `ResourcePath` isn't set.